RecoParticleFlow/PFClusterProducer/interface/PFRecHitQTests.h

```diff
@@ -381,6 +381,83 @@ class PFRecHitQTestHOThreshold : public PFRecHitQTestBase {
 };
 
 //
+//  Quality test that checks threshold as a function of ECAL eta-ring
+//
+#include "Calibration/Tools/interface/EcalRingCalibrationTools.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+class PFRecHitQTestECALThreshold : public PFRecHitQTestBase {
+ public:
+  PFRecHitQTestECALThreshold() {
+
+  }
+
+  PFRecHitQTestECALThreshold(const edm::ParameterSet& iConfig):
```

Comment:
Maybe it would be more intuitive to call this PFRecHitQTestECALMultiThreshold
or PFRecHitQTestECALThresholds